external/implementations/samlconf-openam-impl/README.md

```diff
@@ -0,0 +1,58 @@
+<!--
+Copyright (c) 2018 Codice Foundation
+
+Released under the GNU Lesser General Public License; see
+http://www.gnu.org/licenses/lgpl.html
+-->
+
+# The [OpenAM](https://github.com/OpenIdentityPlatform/OpenAM) SAML Implementation
+## Known Compliance Issues
+| Issue| Section | Specification Snippet
+| -----| ------- | ---------------------
+| Does not use POST messages as the action attribute | Bindings 3.5.4 | The action attribute of the form MUST be the recipient's HTTP endpoint for the protocol or profile using this binding to which the SAML message is to be delivered. The method attribute MUST be "POST". 
+| Does not validate that Relay States are less than or equal to 80 bytes.| Bindings 3.4.3 | RelayState data MAY be included with a SAML protocol message transmitted with this binding. The value MUST NOT exceed 80 bytes.
+| Does not validate the Name ID in an AuthnRequest | Profiles 4.1.4.1 | SAMLProfiles_4_1_4_1: Note that the service provider MAY include a <Subject> element in the request that names the actual identity about which it wishes to receive an assertion. This element MUST NOT contain any <SubjectConfirmation> elements. If the identity provider does not recognize the principal as that identity, then it MUST respond with a <Response> message containing an error status and no assertions.
+| Does not sign Logout Responses | Profiles 4.4.4.2 | The responder MUST authenticate itself to the requester and ensure message integrity, either by signing the message or using a binding-specific mechanism.
```

Comment:
It also fails on the redirect SLO tests

Reply:
For not being signed? 🤔 Cause it's okay to not sign redirect messages. Is there not a signature parameter in the request URL? If so that should be a different spec reference.

Reply:
ahh I see what you mean, yeah it's not failing on the redirect SLO tests for not being signed